x-pack/plugins/siem/public/components/timeline/auto_save_warning/index.tsx

```diff
@@ -48,49 +49,46 @@ const AutoSaveWarningMsgComponent = pure<OwnProps>(
     timelineId,
     updateAutoSaveMsg,
     updateTimeline,
-  }) => (
-    <EuiGlobalToastList
-      toasts={
-        timelineId != null && newTimelineModel != null
-          ? [
-              {
-                id: 'AutoSaveWarningMsg',
-                title: i18n.TITLE,
-                color: 'warning',
-                iconType: 'alert',
-                toastLifeTimeMs: 15000,
-                text: (
-                  <>
-                    <p>{i18n.DESCRIPTION}</p>
-                    <EuiFlexGroup justifyContent="flexEnd" gutterSize="s">
-                      <EuiFlexItem grow={false}>
-                        <EuiButton
-                          size="s"
-                          onClick={() => {
-                            updateTimeline({ id: timelineId, timeline: newTimelineModel });
-                            updateAutoSaveMsg({ timelineId: null, newTimelineModel: null });
-                            setTimelineRangeDatePicker({
-                              from: getOr(0, 'dateRange.start', newTimelineModel),
-                              to: getOr(0, 'dateRange.end', newTimelineModel),
-                            });
-                          }}
-                        >
-                          {i18n.REFRESH_TIMELINE}
-                        </EuiButton>
-                      </EuiFlexItem>
-                    </EuiFlexGroup>
-                  </>
-                ),
-              },
-            ]
-          : []
-      }
-      dismissToast={() => {
-        updateAutoSaveMsg({ timelineId: null, newTimelineModel: null });
-      }}
-      toastLifeTimeMs={6000}
-    />
-  )
+  }) => {
+    const dispatchToaster = useStateToaster()[1];
+    if (timelineId != null && newTimelineModel != null) {
+      const toast: Toast = {
+        id: 'AutoSaveWarningMsg',
+        title: i18n.TITLE,
+        color: 'warning',
+        iconType: 'alert',
+        toastLifeTimeMs: 15000,
```

Comment:
Did you want 15s here? The [toast guidelines](https://elastic.github.io/eui/#/guidelines/toasts) say they should stay on the screen around 10s.